### PR TITLE
Update LESS + app preview toggle for new formplayer

### PR DESCRIPTION
**Pull request by biyeun**
- reorganizes formplayer / cloudcare css for future styling
- toggle for app preview updated so that button is less obstructive (can be moved as the menu design changes)
- adds a formplayer font with future icons mimicking mobile!

@benrudolph cc @orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/preview_app.js [hunk 1]
```diff
@@ -1,14 +1,77 @@
...
+                $('.preview-action-show').removeClass('hide');
+                $('.preview-action-hide').addClass('hide');
+            }
+        };
+        if ($togglePreviewBtn) {
+            $togglePreviewBtn.click(_toggleAppPreview);
+        }
+        $appPreview.find('.btn-preview-close').click(_toggleAppPreview);
+
+        var _resizeAppPreview = function (event) {
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/preview_app.js]: 'event' is defined but never used (no-unused-vars)

**Comment by biyeun** [comment]
scroll handling for vertical window size
scroll

toggle functionality
toggle

**Comment by czue** [comment]
whoa this is looking awesome

File: corehq/apps/style/static/app_manager/less/preview_app-main.less [hunk 2]
```diff
@@ -1,29 +1,76 @@
...
+  .border-bottom-radius(@preview-close-size/2);
+  width: @preview-close-size;
+  height: @preview-close-size;
+  box-sizing: border-box;
+  padding: 0;
+  text-align: center;
+  color: lighten(@cc-text, 60);
+  font-size: 1.6rem;
+  padding-left: 1px;
+  .transition(all .6s);
```
Comment (benrudolph) [inline corehq/apps/style/static/app_manager/less/preview_app-main.less]: nice cleanup 😄

File: corehq/apps/style/static/cloudcare/less/font-formplayer/variables.less [hunk 3]
```diff
@@ -0,0 +1,17 @@
+@font-path: "/static/cloudcare/font";
+@css-prefix: ff;
+@inverse: #ffffff;
+@font-family: 'FormplayerFontRegular';
```
Comment (benrudolph) [inline corehq/apps/style/static/cloudcare/less/font-formplayer/variables.less]: you designed your own font for this??

File: corehq/apps/style/static/app_manager/less/preview_app-main.less [hunk 4]
```diff
@@ -1,6 +1,13 @@
 @import '../../style/less/_hq/includes/variables.less';
 @import '../../style/less/_hq/includes/mixins.less';
 
+/* this file is included in app_manager/apps_base.html and is relevant to the
+EXTERNAL styling of the preview app container in app manager.
+
+For the styling of the app preview content and layout, please check out
+preview_app/less */
```
Comment (benrudolph) [inline corehq/apps/style/static/app_manager/less/preview_app-main.less]: 👍

File: corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html [hunk 5]
```diff
@@ -30,10 +30,22 @@
...
+  <link type="text/less"
+        rel="stylesheet"
+        media="all"
+        href="{% static 'cloudcare/less/cloudcare.debug.less' %}" />
+  {% else %}
+  {% compress css %}
+      <link type="text/less"
+            rel="stylesheet"
+            media="all"
+            href="{% static 'cloudcare/less/cloudcare.less' %}"/>
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html]: so excited for `less` in cloudcare

**Comment by benrudolph** [comment]
👏 👏 👏  this is really nice

File: corehq/apps/style/static/cloudcare/less/font-formplayer/variables.less (hunk 3 above)
Comment (biyeun) [inline corehq/apps/style/static/cloudcare/less/font-formplayer/variables.less]: it's for the icons

**Pull request merged**